ui/app/templates/vault/cluster/secrets/backend/configuration.hbs

```diff
@@ -26,23 +26,28 @@
       </ToolbarLink>
     </ToolbarActions>
   </Toolbar>
-{{/if}}
 
-<div class="box is-fullwidth is-sideless is-paddingless is-marginless">
-  {{#each this.model.attrs as |attr|}}
-    {{#if (eq attr.type "object")}}
-      <InfoTableRow
-        @alwaysRender={{not (is-empty-value (get this.model attr.name))}}
-        @label={{or attr.options.label (to-label attr.name)}}
-        @value={{stringify (get this.model (or attr.options.fieldValue attr.name))}}
-      />
-    {{else}}
-      <InfoTableRow
-        @alwaysRender={{and (not (is-empty-value (get this.model attr.name))) (not-eq attr.name "version")}}
-        @formatTtl={{eq attr.options.editType "ttl"}}
-        @label={{or attr.options.label (to-label attr.name)}}
-        @value={{get this.model (or attr.options.fieldValue attr.name)}}
-      />
-    {{/if}}
-  {{/each}}
-</div>
\ No newline at end of file
+  <SecretEngine::ConfigurableSecretEngineDetails @model={{this.model}} />
```

Comment:
Yep - I commented on this above but seeing this in action I think `SecretEngine::ConfigurationPage` would be super readable and clear here. The conditional above makes it super easy to tell that this component renders if `this.isConfigurable`